controllers/storagecluster/volumegroupsnapshotterclasses.go

```diff
@@ -56,15 +56,19 @@ func newCephFilesystemGroupSnapshotClassConfiguration(instance *ocsv1.StorageClu
 	return GroupSnapshotClassConfiguration{
 		groupSnapshotClass: newVolumeGroupSnapshotClass(instance, cephfsGroupSnapshotter),
 		reconcileStrategy:  ReconcileStrategy(instance.Spec.ManagedResources.CephFilesystems.ReconcileStrategy),
-		disable:            instance.Spec.AllowRemoteStorageConsumers,
+		//TODO: This resource will be managed by the provider-client flow, replacing it with same field used to disable
```

Comment:
open a jira task for the specific TODO so that we dont miss it

Reply:
There is a jira already present and Malay is working on it: https://github.com/red-hat-storage/ocs-operator/pull/3037